### PR TITLE
Update tw_docs.md

### PR DESCRIPTION
**Pull request by dcoles**
Correct typo and slightly rephrase sentence about Face navigation.

### CODE REVIEW

File: Documentation/tw_docs.md [hunk 1]
```diff
@@ -55,4 +55,4 @@ This source code is released under the GPL 3.0 license. Please read the license
 ## Getting Started
 Whether you are a developer looking to extend the functionality of the smartwatch or a user curious about the technology behind the device, this documentation is your gateway to unlocking the full potential of the TinyWATCH S3 firmware.
 
-Let's embark on this journey of exploration and innovation as we delve into the heart of the TinyWATCH S3 firmware.
\ No newline at end of file
+Let's embark on this journey of exploration and innovation as we delve into the heart of the TinyWATCH S3 firmware.
```
Comment (dcoles) [inline Documentation/tw_docs.md:58]: Huh. That's annoying. It seems GitHub's "in-place" editing will append a new-line at the end of the file.

Happy to revert if that's undesired.

Comment (UnexpectedMaker) [reply Documentation/tw_docs.md:58]: There's supposed to be an empty new line at the end. That's common practice.

**Comment by UnexpectedMaker** [comment]
I'll merge this, but don't worry about any more doc fixes for now, they are dramatically behind the current state of the code and need to be ripped out and replaced with something automated (at some stage).

File: Documentation/tw_docs.md (hunk 1 above)
Comment (dcoles) [reply Documentation/tw_docs.md:58]: I agree, I'm just surprised the GitHub editor did so simply by touching the file. Though I sometimes wish there was a standard `git smudge` for it on text files.

**Pull request merged**